cli/src/commands/fee-profile/addForumPost.ts

```diff
@@ -0,0 +1,45 @@
+import FeeProfileCommandBase from '../../base/FeeProfileCommandBase'
+import { flags } from '@oclif/command'
+import _ from 'lodash'
+import { formatBalance } from '@polkadot/util'
+import chalk from 'chalk'
+
+const DEFAULT_POST_LENGTH = 200
+
+export default class FeeProfileAddForumPost extends FeeProfileCommandBase {
+  static description = 'Create fee profile of forum.add_post extrinsic.'
+
+  static flags = {
+    postLen: flags.integer({
+      char: 'p',
+      default: DEFAULT_POST_LENGTH,
+      description: 'Post length to use for estimating tx fee',
+    }),
+    editable: flags.boolean({
+      char: 'e',
+      description: 'If specified - `editable` parameter is set to true when estimating the costs',
+    }),
+    ...super.flags,
+  }
+
+  async run(): Promise<void> {
+    const api = await this.getOriginalApi()
```

Comment:
Here `await` is unnecessary. Also, in all the fee-profile commands, the same expression with await is used. Can you please have a look at all these instances
```suggestion
    const api = this.getOriginalApi()
```